.gitignore

```diff
@@ -4,7 +4,6 @@
 # pycache
 *__pycache__*
 .nfs*
-.vscode
 
```

Comment:
```suggestion
.vscode/launch.json
.vscode/settings.json
```

Reply:
I would like to keep these. I use them to configure unittests in vscode

Reply:
I would like to _not_ keep it, so contributors automatically get a yaml validator! ;)

Reply:
I learned that there are 3 types of settings now in vscode:  User, Remote and Workspace. This change will impact workspace settings, but I can still use my user settings to enable test. This is great! So this change will apply workspace settings.